.github/workflows/ci.yml

```diff
@@ -1,22 +1,24 @@
-name: Node.js CI
-on: [push, pull_request]
+name: CI
+on:
+  - push
+  - pull_request
 jobs:
-  build:
+  test:
+    name: Node.js ${{ matrix.node-version }} on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
```

Comment:
If one job fails (i.e. Node.js 14 on Windows) then still continue running the rest of the tests